galpy/potential/amuse.py

```diff
@@ -179,9 +188,10 @@ def circular_velocity(self,r):
         HISTORY:
            2019-08-12 - Written - Webb (UofT)
         """
-        return potential.vcirc(self.pot,r.value_in(units.kpc)/self.ro,phi=0,
-                               t=self.tgalpy,ro=self.ro,vo=self.vo) | units.kms
-        
+        return potential.vcirc(self.pot,r.value_in(u.kpc)/self.ro,phi=0,
```

Comment:
I think this needs to be multiplied by ``self.vo`` to work properly (right now the galpy output is in internal units and is then given AMUSE kms units)

Reply:
yes. having `use_physical=False` and multiplying by `self.vo` appears to work.

Reply:
I've implemented this in a new commit.